### PR TITLE
Refactor UI

### PR DESCRIPTION
**Pull request by yimjianbing**
fixes #93

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/CommandResult.java [hunk 1]
```diff
@@ -16,15 +16,19 @@ public class CommandResult {
...
     private final boolean exit;
 
     /**
      * Constructs a {@code CommandResult} with the specified fields.
      */
-    public CommandResult(String feedbackToUser, boolean showHelp, boolean exit) {
+    public CommandResult(String feedbackToUser, boolean showHelp, boolean showGroups, boolean exit) {
         this.feedbackToUser = requireNonNull(feedbackToUser);
         this.showHelp = showHelp;
+        this.showGroups = showGroups;
```
Comment (wenjebs) [inline src/main/java/seedu/address/logic/commands/CommandResult.java:31]: could be an object for easier understanding? was confused when used this was created in other classes

File: src/main/java/seedu/address/logic/commands/ExitCommand.java [hunk 2]
```diff
@@ -13,7 +13,7 @@ public class ExitCommand extends Command {
 
     @Override
     public CommandResult execute(Model model) {
-        return new CommandResult(MESSAGE_EXIT_ACKNOWLEDGEMENT, false, true);
+        return new CommandResult(MESSAGE_EXIT_ACKNOWLEDGEMENT, false, false, true);
```
Comment (wenjebs) [inline src/main/java/seedu/address/logic/commands/ExitCommand.java:16]: confusion here as mentioned in CommandResult constructor

File: src/main/java/seedu/address/ui/GroupsWindow.java [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+     *             if this method is called during animation or layout processing.
+     *         </li>
+     *         <li>
+     *             if this method is called on the primary stage.
+     *         </li>
+     *         <li>
+     *             if {@code dialogStage} is already showing.
+     *         </li>
+     *     </ul>
+     */
```
Comment (wenjebs) [inline src/main/java/seedu/address/ui/GroupsWindow.java:65]: remember to remove large chunks of commented code

**Review by wenjebs — COMMENTED** [review]
lgtm, fix styles

File: src/main/java/seedu/address/ui/GroupCard.java [hunk 4]
```diff
@@ -34,10 +34,6 @@ public class GroupCard extends UiPart<Region> {
     @FXML
     private Label id;
     @FXML
-    private Label studentClass;
```
Comment (Hoodini231) [inline src/main/java/seedu/address/ui/GroupCard.java]: Should we keep these in the cards even under group ?

File: src/main/resources/view/DarkTheme.css [hunk 5]
```diff
@@ -350,3 +350,6 @@
     -fx-background-radius: 2;
     -fx-font-size: 11;
 }
+
```
Comment (Hoodini231) [inline src/main/resources/view/DarkTheme.css:353]: Excessive lines here prehaps?

**Review by Hoodini231 — APPROVED** [review]
LGTM other than the failing style check rn

File: src/main/resources/view/DarkTheme.css (hunk 5 above)
Comment (yimjianbing) [reply src/main/resources/view/DarkTheme.css:354]: There needs to be a new line an end of file if not the checkstyle will fail

**Pull request merged**